packages/styles/pagination.css

```diff
@@ -0,0 +1,71 @@
+.Pagination > ul {
+  list-style-type: none;
+  display: flex;
+  align-items: center;
+}
+
+.Pagination > ul > li {
+  margin-right: 5px;
+}
+
+.Pagination button {
+  padding: var(--space-three-quarters);
+  height: 36px;
+  width: 36px;
```

Comment:
Nitpick: Button height should scale with user set font-size, so rems would be preferable here.

Reply:
good catch